webex/content.js

```diff
@@ -129,43 +128,37 @@ function setText(id, text) {
     fadeBackground(e);
 }
 
+function getActiveElement(element = document.activeElement) {
+    if (element.nodeName === 'IFRAME' || element.nodeName === 'FRAME')
+        return null; // skip to allow child documents to handle shortcut
+
+    const shadowRoot = element.shadowRoot
+    const contentDocument = element.contentDocument
+
+    if (shadowRoot && shadowRoot.activeElement) {
+        return getActiveElement(shadowRoot.activeElement)
+    }
```

Comment:
Interesting, didn't know about shadow roots before. Do you have an example page where a textarea on which you'd want to use Textern is actually nested under a shadow root?

Reply:
No, it was only to ensure we attach to the right element. I read [this answer on SO](https://stackoverflow.com/a/61167135), and didn't want neglect pages that used more complex DOMs.